TLM/TLM/Custom/PathFinding/CustomPathManager.cs

```diff
@@ -106,14 +181,14 @@ public class CustomPathManager : PathManager {
 			Log.Warning($"CustomPathManager.ReleasePath({unit}) called.");
 #endif
 
-            if (m_pathUnits.m_buffer[unit].m_simulationFlags == 0) {
+            if(m_pathUnits.m_buffer[unit].m_simulationFlags == 0) {
```

Comment:
disappearing spaces everywhere. I thought we had `.editorconfig` rules for that? `if <space> (`